torch/testing/_internal/common_methods_invocations.py

```diff
@@ -89,16 +105,21 @@ def get_inplace(self):
         """
         return self.inplace_variant
 
+    def sample_inputs(self, device, dtype, requires_grad=False):
+        """Returns an iterable of SampleInputs."""
+        return tuple()
```

Comment:
Should this be a `NotImplementedError`? Or are there cases where it is valid to not override it?

Reply:
Having this throw an error is reasonable but would require tests catch that error. Returning a tuple lets the case of no sample inputs be handled gracefully. It would protect us, however, from people implementing OpInfos and not adding sample inputs. If that becomes an issue we may want to revisit this decision.